docs/algorithm.md

```diff
@@ -51,15 +51,15 @@ This is an example of a rule definition.
         },
         {
             "type": "direct-translation",
```

Comment:
What do you think change from `direct-translation` to `dynamic`

Reply:
Also, I was thinking on that `literal` should be exactly what it means. We should add the value passed without any changes. And the rest will fill in the `dynamic`. 

:)

Reply:
I thought the same, to use `literal` instead of `direct-translation`, but I thought too that the expression `direct-translation` help to understand in a easy way what you want to do, but the concept is different, in that you are right.

We can use the type `dynamic` for other things, at this moment, it doesn't have any use because we are resolving all through templates.

If you prefer, no problem, I can change it to literal.

Reply:
Yes, I think it's better follow a simple and concrete semantic. :)